modules/vstudio/tests/vc2019/test_compile_settings.lua

```diff
@@ -106,3 +106,27 @@
 	<ScanSourceForModuleDependencies>false</ScanSourceForModuleDependencies>
 		]]
 	end
+
+	function suite.UseStandardPreprocessorOn()
+		usestandardpreprocessor 'on'
```

Comment:
You have the API as "On" and "Off", then you're using it as "on" and "off" here. Is it case agnostic?  If so, can we add a test for each case (just as a sanity check).